cmd/cli/namespace_add.go

```diff
@@ -56,14 +68,15 @@ func newNamespaceAdd(out io.Writer) *cobra.Command {
 			namespaceAdd.clientSet = clientset
 			return namespaceAdd.run()
 		},
+		Example: namespaceAddExample,
 	}
 
 	//add mesh name flag
 	f := cmd.Flags()
 	f.StringVar(&namespaceAdd.meshName, "mesh-name", "osm", "Name of the service mesh")
 
 	//add sidecar injection flag
-	f.BoolVar(&namespaceAdd.enableSidecarInjection, "enable-sidecar-injection", true, "Enable automatic sidecar injection")
+	f.BoolVar(&namespaceAdd.enableSidecarInjection, "enable-sidecar-injection", true, "Enable or disable automatic sidecar injection, disabling requires explicitly setting this flag.")
```

Comment:
If my understanding is correct, this may help clarify a bit:
```suggestion
	f.BoolVar(&namespaceAdd.enableSidecarInjection, "enable-sidecar-injection", true, "Enable or disable automatic sidecar injection, disabling requires explicitly setting this flag to false.")
```